pkg/config/process.go

```diff
@@ -69,4 +93,25 @@ func setupProcesses(config Config) {
 		"DD_PROCESS_AGENT_DISCOVERY_ENABLED",
 	)
 	procBindEnvAndSetDefault(config, "process_config.process_discovery.interval", 4*time.Hour)
+
+	AddOverrideFunc(loadProcessTransforms)
+}
+
+// loadProcessTransforms loads transforms associated with process config settings. This is used to handle deprecated settings
+func loadProcessTransforms(config Config) {
+	if config.IsSet("process_config.enabled") {
+		log.Debug("process_config.enabled is deprecated, use process_config.container_collection.enabled" +
+			" and process_config.process_collection.enabled instead")
+		procConfigEnabled := strings.ToLower(config.GetString("process_config.enabled"))
+		if procConfigEnabled == "disabled" {
+			config.Set("process_config.process_collection.enabled", false)
+			config.Set("process_config.container_collection.enabled", false)
+		} else if enabled, _ := strconv.ParseBool(procConfigEnabled); enabled { // "true"
+			config.Set("process_config.process_collection.enabled", true)
+			config.Set("process_config.container_collection.enabled", false)
+		} else { // "false"
```

Comment:
Do I understand this right: `disabled` and `false` have different meaning ? If so, I think we should explain it in the releasenote so users know how to use the 2 new settings.

Reply:
Yep, unfortunately you are understanding this correctly. `process_config.enabled` has 3 possible values:
- `true`: Enables process check (which also collects containers)
- `false`: Enables container check
- `disabled`: Disables process and container collection

I think it makes sense to explain the new equivalent settings in the release note. I'll go ahead and add it